packages/components/psammead-brand/src/index.jsx

```diff
@@ -71,6 +71,8 @@ const brandWidth = (minWidth, maxWidth) => `
   width: 100%;
   max-width: ${maxWidth / 16}rem;
   min-width: ${minWidth / 16}rem;
+  flex: 1 1 ${minWidth / 16}rem;
+  -ms-flex-preferred-size: ${maxWidth / 16}rem;
```

Comment:
The update to `flex` works well on supported browsers. 
For `-ms-flex-preferred-size`, I looked on Browserstack with IE11 and removed this style and did not see any change in behaviour. Am I missing something here?

Reply:
<img width="524" alt="image" src="https://user-images.githubusercontent.com/43134742/72157819-bad4f280-33b0-11ea-87ba-bcaae8ee94f1.png">
<img width="117" alt="image" src="https://user-images.githubusercontent.com/43134742/72157861-cd4f2c00-33b0-11ea-9ca8-c6131d4baeab.png">

This is what I see with `-ms-flex-preferred-size` turned off on IE11 - the script switch is off the edge of the screen.

Reply:
Seems like I had an issue with the changes in the browserstack dev tools not applying correctly. When I made changes locally I see the same as your screenshot.  
So, I'm happy with the change made!